api/types.go

```diff
@@ -161,6 +166,35 @@ type (
 		CallbackURL string   `json:"callbackURL"`
 		Scopes      []string `json:"scopes"`
 	}
+
+	// TPoolResp is the response body for the [GET] /tpool/fee endpoint
+	TPoolResp struct {
+		RecommendedFee types.Currency
+	}
+
+	// AccountResp is the response body for the [GET] /accounts endpoint
+	AccountResp []accounts.Account
+
+	// VolumeResp is the response body for the [GET] /volumes endpoint
+	VolumeResp []VolumeMeta
+
+	// AlertResp is the response body for the [GET] /alerts endpoint
+	AlertResp []alerts.Alert
+
+	// PeerResp is the response body for the [GET] /syncer/address endpoint
+	PeerResp []Peer
+
+	// SyncAddrResp is the response body for the [GET] /syncer/peers endpoint
+	SyncAddrResp modules.NetAddress
```

Comment:
This should be `string` not `modules.NetAddress`

Reply:
changing to string I no longer get a response with `?response=prometheus` for this endpoint

Reply:
```
ERROR       api     failed to marshal prometheus response   {"error": "failed to encode metric: unsupported label value api.SyncAddrResp"}
```

Reply:
https://github.com/SiaFoundation/hostd/pull/295#discussion_r1482300804